openapi/rhose-api.yaml

```diff
@@ -895,30 +1002,38 @@ components:
         total:
           format: int64
           type: integer
-    ErrorResponse:
+    ErrorsResponse:
       type: object
       properties:
         kind:
           type: string
-        id:
-          type: string
-        name:
-          type: string
-        href:
-          type: string
-        code:
-          type: string
-        reason:
-          type: string
-    ErrorsResponse:
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/Error'
+    List:
```

Comment:
I do not see us actually using list anywhere? The point is to return List for every non plural response. 
We require it only because that will give us out of the box migration to KCP API thru SDK generator.

Reply:
This `List` openapi component is a consequence of my changes in `PagedListResponse.java`, especially:
```java
@Schema(name = "List",
        allOf = { PagedListResponse.class, ListResponse.class })
public abstract class PagedListResponse<T> extends ListResponse<T> {
```

The issue is, without the changes above, the `spectral lint` check reports this error:
```
  821:11    error  rhoas-list-schema              Missing required schema object "List"                                                           components.schemas
```

@wtrocki so what is your advice please?

Reply:
see my changes in https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/861/commits/f267fec87ce979bcaba4280bdf61827f433ad453